spec/util/subscribeToResult-spec.ts

```diff
@@ -0,0 +1,156 @@
+import {expect} from 'chai';
+import * as Rx from '../../dist/cjs/Rx';
+import {subscribeToResult} from '../../dist/cjs/util/subscribeToResult';
+import {OuterSubscriber} from '../../dist/cjs/OuterSubscriber';
+import {$$iterator} from '../../dist/cjs/symbol/iterator';
+import $$symbolObservable from 'symbol-observable';
+
+describe('subscribeToResult', () => {
+  it('should immeidately complete when subscribe to scalarObservable', (done: MochaDone) => {
+    const result = Rx.Observable.of(42);
+    const subscriber = new OuterSubscriber(x => {
+      expect(x).to.be.equal(42);
+    }, () => {
+      done(new Error('should not be called'));
+    }, done);
```

Comment:
There's no guarantee made by this test that it has synchronously completed.

You'll want to not use `done` here, but rather assert on the line immediately after the `subscribeToResult` call. (line 18 or 19, etc). Make the test synchronous.